docs/specs/aerogear-client-pipe/usage-examples.asciidoc

```diff
@@ -1,5 +1,6 @@
 ---
```

Comment:
This is likely replaced by `docs/specs/aerogear-client-paging-usage/index.markdown`